Kentico.Kontent.Delivery.Tests/Extensions/FakeServiceCollection.cs

```diff
@@ -1,172 +0,0 @@
-﻿using System;
-using System.Collections;
-using System.Collections.Generic;
-using System.Linq;
-using Kentico.Kontent.Delivery.Abstractions.InlineContentItems;
-using Kentico.Kontent.Delivery.InlineContentItems;
-using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Options;
-
-namespace Kentico.Kontent.Delivery.Tests.Extensions
-{
-    /// <summary>
-    /// Mock ServiceCollection that keeps inside everything that was registered.
-    /// Can be used to check whether expected instances were added to the ServiceCollection.
-    /// </summary>
-    internal class FakeServiceCollection : IServiceCollection
```

Comment:
why did you decide to remove it?

Reply:
With FaceServiceCollection implementation is more probably we can do a mistake with this "faking" implementation. We need more realistic tests with ServiceCollection. I did some basic scenarios with it. In the future, we can do more scenarios

Reply:
ok